packages/pagination/src/CurrentPage.tsx

```diff
@@ -0,0 +1,44 @@
+import { classNames } from "@chbphone55/classnames";
+import { i18n } from '@lingui/core';
+import { pagination as ccPagination } from '@warp-ds/css/component-classes';
+import { usePagination } from './PaginationContainer.js';
+import React from 'react';
+
+type CurrentPageProps = {
+  /**
+   * Label to render on mobile
+   */
+  children?: React.ReactNode;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+const CurrentPage = React.forwardRef<HTMLDivElement, CurrentPageProps>(
+  ({ children, className, ...props }, ref) => {
+    const { currentPage } = usePagination();
+
+    if (currentPage <= 1) {
+      return null;
+    }
```

Comment:
I think a component should always render something - I would move this condition to the parent instead, so it's apparent already there what will and what won't be rendered. Wdyt?